src/attached_probe.cpp

```diff
@@ -393,19 +393,14 @@ static void check_alignment(std::string &path,
 
   std::string tmp = path + ":" + symbol + "+" + std::to_string(func_offset);
 
-  // If we did not allow unaligned uprobes in the
-  // compile time, force the safe mode now.
-#ifndef HAVE_UNSAFE_PROBE
-  safe_mode = true;
-#endif
-
   switch (aligned) {
     case AlignState::Ok:
       return;
     case AlignState::NotAlign:
       if (safe_mode)
         LOG(BUG) << "Could not add " << type
```

Comment:
not actually a bug right? LOG(BUG) triggers coredump, probably want to throw `FatalUserException`.

probably in a different PR though

Reply:
Ah true. Good catch. Maybe same PR, but a separate commit.